types/components/Chip.d.ts

```diff
@@ -2,17 +2,17 @@ import * as React from "react";
 import { AutocompleteOptions } from "./Autocomplete";
 import { AnyFn, SharedBasic } from "./utils";
 
-interface ChipItem {[key: string]: string;}
+export interface ChipItem {[key: string]: string;}
 
 export interface ChipOptions {
   data?: ChipItem[];
   placeholder?: string;
   secondaryPlaceholder?: string;
   autocompleteOptions?: AutocompleteOptions;
   limit?: number;
-  onChipAdd?: AnyFn;
-  onChipSelect?: (item: React.ReactNode) => void;
-  onChipDelete?: AnyFn;
+  onChipAdd?: (item: JQuery, chip: HTMLElement) => void;
+  onChipSelect?: (item: JQuery, chip: HTMLElement) => void;
+  onChipDelete?: (item: JQuery, chip: HTMLElement) => void;
```

Comment:
`Cannot find name 'JQuery'.`
I would rather leave this as `any` since JQuery is not required to use this package anymore

Reply:
Also, doesn't `React.ReactNode` cover the case of `HTMLElement`? Why this change?

Reply:
I'm not entirely sure - i just needed it to accept to params to unblock me.  without the second param there's no way of telling what was selected.

Reply:
let's use any instead of JQuery please